manifests/install.pp

```diff
@@ -218,8 +218,16 @@
     'FreeBSD': {
       # repo is not applicable to windows
     }
+    'Gentoo': {
+      file { '/etc/portage/package.accept_keywords/telegraf':
+        ensure  => file,
+        owner   => 'root',
+        group   => 'root',
+        content => 'net-analyzer/telegraf ~amd64',
```

Comment:
I'm not sure if we should always allow testing for this package. maybe restrict it to the current 1.18.3 package? And this file needs a newline at the end:
```suggestion
        content => "net-analyzer/telegraf ~amd64\n",
```